README.md

```diff
@@ -61,6 +61,57 @@ If you want your mail server to scan messages for viruses and spam, you will hav
 # apt-get install clamav-daemon clamav-freshclam spamassassin
 ```
 
+
+**EL:** - For RedHat based (Enterprise Linux, CentOS, RHEL, Scientific) Exim is not build linked against Mysql libraries making it imposible to use vexim against a RDBMS such as mysql or postfix. The EPEL project provides support for Exim that will utilize Mysql. The EPEL repo is a Red Hat sponsored community based repo, details can be found https://fedoraproject.org/wiki/EPEL
+
+We first must enable the EPEL repo for the right version of Enterprise Linux. Your host will need outgoing access to the internet on port 443.
+for Enterprise Linux 6.X
+```
+rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+yum update
+
+``` 
+For Enterprise Linux 7.X
+```
+rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
```

Comment:
Yes must be a cut and paste error, odd I missed it in the read back, I'll fix and resubmit 

> On 27 Jul 2016, at 9:09 pm, Rimas Kudelis <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -61,6 +61,57 @@ If you want your mail server to scan messages for viruses and spam, you will hav
> >  # apt-get install clamav-daemon clamav-freshclam spamassassin
> >  ```
> >  
> > +
> > +**EL:** - For RedHat based (Enterprise Linux, CentOS, RHEL, Scientific) Exim is not build linked against Mysql libraries making it imposible to use vexim against a RDBMS such as mysql or postfix. The EPEL project provides support for Exim that will utilize Mysql. The EPEL repo is a Red Hat sponsored community based repo, details can be found https://fedoraproject.org/wiki/EPEL
> > +
> > +We first must enable the EPEL repo for the right version of Enterprise Linux. Your host will need outgoing access to the internet on port 443.
> > +for Enterprise Linux 6.X
> > +```
> > +rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
> > +yum update
> > +
> > +``` 
> > +For Enterprise Linux 7.X
> > +```
> > +rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
> This is exactly the same line as above. Typo?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>